Dockerfile

```diff
@@ -2,7 +2,12 @@ FROM google/cloud-sdk:latest
 
 RUN apt-get update
 RUN apt-get --yes install openjdk-11-jre
+RUN apt-get --yes install samtools
```

Comment:
These can be combined in 1 line to make things a little more concise.

I would probably fix a version here, as the version could change implicitly when someone builds a new image from this dockerfile.